testdata/htsjdk/samtools/cram/test.dict

```diff
@@ -1,2 +1,3 @@
 @HD	VN:1.4	SO:unsorted
 @SQ	SN:Sheila	LN:20	M5:7ddd8a4b4f2c1dec43476a738b1a9b72	UR:file:/Users/edwardk/Documents/htsjdk/testdata/htsjdk/samtools/cram/auxf.fa
+@SQ	SN:chr1	LN:8	M5:cc0af3a4fedb18378b4b57b98068e69f
```

Comment:
As discussed in person, the reference here should not have the contig Sheila in order to trigger your new check (your test case was not actually being run in travis due to the test class being in the wrong place)